Library/Homebrew/tap.rb

```diff
@@ -112,6 +112,16 @@ def to_hash
     }
   end
 
+  def load_tap_formula_renames
+    begin
+      path.cd do
+        load "tap_formula_renames.rb"
```

Comment:
I used `tap_formula_renames.rb` because the name is pretty similar to `formula_renames.rb` in core repository.